src/CoreFx.Private.TestUtilities/src/CoreFx.Private.TestUtilities.csproj

```diff
@@ -7,7 +7,6 @@
     <ShouldWriteSigningRequired>false</ShouldWriteSigningRequired>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnablePInvokeAnalyzer>false</EnablePInvokeAnalyzer>
-    <NoWarn>$(NoWarn);CS1572;CS1574;CS1587;CS1589;CS1591;CS1723;CS1734</NoWarn>
```

Comment:
```
System\PlatformDetection.Unix.cs(185,26): error CS1572: XML comment has a param tag for 'versionId', but there is no parameter by that name [D:\j\workspace\windows-TGrou---0d2c9ac4\src\CoreFx.Private.TestUtilities\src\CoreFx.Private.TestUtilities.csproj]
System\PlatformDetection.Unix.cs(196,26): error CS1572: XML comment has a param tag for 'versionId', but there is no parameter by that name [D:\j\workspace\windows-TGrou---0d2c9ac4\src\CoreFx.Private.TestUtilities\src\CoreFx.Private.TestUtilities.csproj]
```

Reply:
Thank you, I will check Linux and OSX builds separately because I was not able to build them locally,.

Reply:
I get strange error: `C:\corefx\external\runtime\runtime.depproj(94,5): error MSB3073: The command "chmod +x C:\corefx\artifacts\bin\testhost/netcoreapp-Linux-Debug-x64/dotnet" exited with code 9009.`
The command is: `build /p:os=OSX`

Reply:
Try:
build /p:OSGroup=OSX
or
build /p:OSGroup=Linux

Reply:
Thank you, it worked. Maybe we can add it to [Advanced Build Options](https://github.com/dotnet/corefx/wiki/Build-and-run-tests#advanced-build-options) or I just wasn't able to find the right doc for it.
I will fix all Linnix / OSX / uap issues.